src/Workspaces/CSharp/Portable/Extensions/ArgumentSyntaxExtensions.cs

```diff
@@ -105,9 +105,17 @@ public static RefKind GetRefKind(this ArgumentSyntax argument)
             SemanticModel semanticModel,
             CancellationToken cancellationToken)
         {
+            TypeInfo typeInfo;
```

Comment:
I assume you are talking about the change in GetDelegatingConstructor. It is nearly identical, but not quite the same. I tried to keep the code as close as possible to the original behavior. If you are fine with changing GetDelegatingConstructor to use DetermineParameterType, I will make the change.

---
In reply to: [68112952](https://github.com/dotnet/roslyn/pull/12159#discussion_r68112952) [](ancestors = 68112952)